src/Utilities/useConnectionStatus.js

```diff
@@ -28,7 +28,9 @@ export const useConnectionStatus = (remediation) => {
           setTotalSystems(totalSystemsCount);
       } catch (error) {
         console.error(error);
-        setDetailsError(error.errors[0].status);
+        setDetailsError(error?.errors[0].status || '');
```

Comment:
when the backend returns a 502 error, it doesnt actually send a body in the response. This is just so that if that error is hit, the page doesnt explode trying to locate it. For other errors, a body is returned

Reply:
We don't need no stinking' body!....   (also: gross, but this will work for now.  I'll add this to the list of things to fix in the api...)